Stripe/PublicHeaders/STPIntentActionDisplayOXXODetails.h

```diff
@@ -0,0 +1,43 @@
+//
+//  STPIntentActionDisplayOXXODetails.h
+//  Stripe
+//
+//  Created by Polo Li on 6/23/20.
+//  Copyright © 2020 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "STPAPIResponseDecodable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ Contains OXXO details necessary for the customer to complete the payment.
+ */
+
```

Comment:
nit delete this space, might mess with doc autogen

Reply:
fixed in https://github.com/stripe/stripe-ios/pull/1592/commits/d1119ead20214924cf571135071f9869cfc0d41b